components/storage/blockdevice/COMPONENT_SPIF/SPIFBlockDevice.cpp

```diff
@@ -936,7 +936,10 @@ int SPIFBlockDevice::_reset_flash_mem()
                 tr_error("Sending RST failed");
                 status = -1;
             }
-            _is_mem_ready();
+            if (false == _is_mem_ready()) {
```

Comment:
Style barf (backwards and silly boolean comparison instead of just `!`), but I guess it's consistent with other code nearby.

Reply:
Coverity has this nice section which I used to just keep the code consistent:
```
Examples where return value from this function is checked
--
A1 example_checked | SPIFBlockDevice.cpp:982 
B1 example_checked | SPIFBlockDevice.cpp:182 
C1 example_checked | SPIFBlockDevice.cpp:410 
D1 example_checked | SPIFBlockDevice.cpp:324
```